psh/test-touch-rootfs.py

```diff
@@ -0,0 +1,32 @@
+# Phoenix-RTOS
+#
+# phoenix-rtos-tests
+#
+# psh touch command test for rootfs targets
+#
+# Copyright 2022 Phoenix Systems
+# Author: Damian Loewnau
+#
+# This file is part of Phoenix-RTOS.
+#
+# %LICENSE%
+
+import psh.tools.psh as psh
+from psh.tools.common import assert_timestamp_change
+
+
+def assert_hardlinks(p):
+    uptimes = {}
+    # all psh commands are hardlinks
+    psh_cmds = psh.get_commands(p)
+    for psh_cmd in psh_cmds:
+        uptimes[psh_cmd] = psh.uptime(p)
```

Comment:
IMHO invalid, the times would be set to system time - not uptime (so if somebody/something would change the system time - eg. `date` test run previously) this test would fail even though the functionality might work correctly.

Please at least name it `dates` not to confuse it with uptime value in the future (applies elsewhere also)